data/troubleshooting.xml

```diff
@@ -129,7 +129,7 @@ Logging already initialized. Skipping...
             <para>to a port that is not used, e.g. "8899":</para>
             <synopsis format="linespecific">&lt;Set name="port"&gt;&lt;SystemProperty name="jetty.port" default="8899"/&gt;&lt;/Set&gt;</synopsis>
             <para>eXist-db uses port 8443 for confidential communication. Another service may also
-                be using this port. To make eXist-db use another port, open the file <filename moreinfo="none">/tools/jetty/etc/jetty.xml</filename> and change the value
+                be using this port. To make eXist-db use another port, open the file <filename moreinfo="none">/tools/jetty/etc/jetty-http.xml</filename> and change the value
```

Comment:
@clickr thank you very much for your PR. I appreciate you trying to improve our documentation, however, this change isn't quite correct...

The file for `https` is - `tools/jetty/etc/jetty-ssl.xml` and the line you must change is:

```xml
<Set name="port"><Property name="jetty.ssl.port" deprecated="ssl.port"><Default><SystemProperty name="jetty.ssl.port" deprecated="ssl.port" default="8443"/></Default></Property></Set>
```

Would you be able to revise?

Also it might be worth adding if you want to change the http port, the file is: `tools/jetty/etc/jetty-http.xml` and the line to change is:

```xml
<Set name="port"><Property name="jetty.http.port" deprecated="jetty.port"><Default><SystemProperty name="jetty.port" default="8080"/></Default></Property></Set>
```